woocommerce/woocommerce-gutenberg-products-block
# Add Storybook entries for `TextInput`, `ValidatedTextInput`, and `ValidationInputError`

## What

Adds Storybook entries for `TextInput`, `ValidatedTextInput`, and `ValidationInputError`.

Fixes #11689

## Why


To showcase the component and provide a level of documentation.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Run `npm run storybook` and go to `localhost:6006`
2. Check the `External Components/ValidationInputError`, `External Components/ValidatedTextInput` and `External Components/TextInput` stories.
3. Ensure the controls work as expected.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Skipping.
